pkg/utils/flow_utils.go

```diff
@@ -31,22 +31,15 @@ const (
 // 2 is from host to network and 3 is from network to host.
 // ts is the timestamp in nanoseconds.
 func ToFlow(
+	l *log.ZapLogger,
```

Comment:
is there any extra reasoning to passing logger here besides warn? this was pretty concise, requires refactoring usage downstream just for a warn

Reply:
There is no reason to pass the logger here. From some of the pprof we collected, this actually incurred quite a bit of memory usage.